ibm_catalog.json

```diff
@@ -48,16 +48,19 @@
           },
           "iam_permissions": [
             {
-              "role_crns": ["crn:v1:bluemix:public:iam::::role:Editor"],
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
               "service_name": "all-account-management-services"
             },
             {
-              "role_crns": ["crn:v1:bluemix:public:iam::::role:Editor"],
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
               "service_name": "lakehouse"
```

Comment:
Please change the service name to `watsonx-orchestrate`. This is a bug and Issue #80  is created. With this change the fix can be provided earlier.
`lakehouse` is for [watsonx-data](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/dbd034e767ef07baa65b2d0b220ff21e4d5d1cb2/main.tf#L63)